docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -430,28 +474,29 @@ Finally, the following hooks are available for querying set size:
 * `useSsmSetCountsQuery`
 * `useCaseSetCountsQuery`
 
-## Creating a cohort
+## Creating a Cohort
 
-Depending on what your application does, you may want to create a new cohort. Although the GDC Portal SDK provides a
-number of functions for creating a new cohort. It is highly recommended that the application use the provided Button and
-SaveCohortModal components to create a new cohort. The Button and SaveCohortModal components are located in
+Depending on the application function, if may be beneficial to create a new cohort. Although the GDC Data Portal SDK provides a
+number of functions for creating a new cohort. It is highly recommended that the application use the provided `Button` and
+`SaveCohortModal` components to create a new cohort. The `Button` and `SaveCohortModal` components are located in
 the `@gff/portal-proto` package.
 
 To create a cohort using the SaveCohortModal component the following code can be used:
 In summary, the above code flow is:
 
-1. The ProjectsCohortButton component renders a button with the label "Save New Cohort".
-2. When the button is clicked, it sets the state variable `showSaveCohort` to true, which triggers the rendering of the `SaveCohortModal` component.
+1. The `ProjectsCohortButton` component renders a button with the label "Save New Cohort"
+2. When the button is clicked, it sets the state variable `showSaveCohort` to true, which triggers the rendering of
+   the `SaveCohortModal` component.
 3. The `SaveCohortModal` component passed:
-   * an onClose function that sets the showSaveCohort state variable to false.
-   * a filters prop, which is an object defining the filters for the cohort based on the selected projects.
-4. The `SaveCohortModal` will use the passed filter to create, name and save the cohort when the save button is clicked.
+   * An onClose function that sets the showSaveCohort state variable to false.
+   * A filters prop, which is an object defining the filters for the cohort based on the selected projects.
```

Comment:
do we need to surround "filters" with backticks? -- wasn't sure if it's the actual name of the prop

Reply:
Yes. I will add that.